gcovr/tests/text-output/Makefile

```diff
@@ -0,0 +1,17 @@
+all:
+	$(CXX) -fprofile-arcs -ftest-coverage -fPIC main.cpp -o testcase
+
+run: txt xml
+
+txt:
+	./testcase
+	$(GCOVR) -d --txt coverage.txt --xml coverage.xml
```

Comment:
At first I didn't notice that this would create both TXT and XML reports. Could this be clarified with a comment, or maybe made clearer by using a single Makefile target for both like `txt xml:`?

Reply:
Sure. Changed to one target and added a comment.